src/mixins/widget.js

```diff
@@ -31,9 +31,7 @@ it's not the recommended way of making custom components.
 
 If you want to disable this message, pass { connector: true } to the mixin.
 
-(TODO: update v2 link)
-Read more on using connectors: https://community.algolia.com/vue-instantsearch/getting-started/custom-components.html
-        `
+Read more on using connectors: https://alg.li/vue-custom`
```

Comment:
Should this be part of this PR?

Also, why using a URL shortener here and not the actual link?

Reply:
because the real link (not the one here) was really long. Could technically be a different PR, but it isn't really that relevant

Reply:
Right, if we want to make the link a bit more explicit, `https://alg.li/vue-custom-components` could be another name.